jobsdb/jobsdb.go

```diff
@@ -2526,7 +2546,7 @@ func (jd *HandleT) isEmptyResult(ds dataSetT, workspace string, stateFilters, cu
 			return false
 		}
 
-		pVals := []string{}
+		var pVals []string
```

Comment:
[question] how is one better than the other (just so that I keep in mind for future purposes)

Reply:
`var pVals []string` doesn't initialise the slice. Thus doesn't allocate memory. The major reason to use `var pVals []string` is that is more idiomatic. 

To be honest I am more used to doing `pVals := []string{}`, but I try to follow the idiomatic way.

Reply:
Yes @lvrach is correct. With `var pVals []string` there is no allocation but both `len` and `append` would work as expected so you don't have to worry about that.